pkg/controller/traceflow/controller.go

```diff
@@ -50,8 +50,9 @@ const (
 
 	// Min and max data plane tag for traceflow. dataplaneTag=0 means it's not a Traceflow packet.
 	// dataplaneTag=15 is reserved.
-	minTagNum uint8 = 1
-	maxTagNum uint8 = 14
+	minTagNum uint8 = 1*4 + 3
```

Comment:
could you add comments for the bits? It is not easy for other developers to understand why it's like this.
Like the meaning of 0bxxxx11, and an example for minTag 0b000111.

Reply:
and the comment above is no longer valid I think

Reply:
Updated comments and formats here.